DMTR-51.tex

```diff
@@ -163,31 +180,207 @@ \subsection{Overall Assessment \label{sect:overallassessment}}
+  }
+
+  \item{
+    Making what can seem like minor changes to the pipeline definition can require more integration time.
+    Examples of pipeline changes that affect the pipeline execution configuration and verification tests:
+
+    \begin{itemize}
+
+      \item{Pipeline code changes or task configuration changes can alter the required input files or what output files are created.}
+      \item{Changing what pipeline steps should be executed may require a pipeline task configuration change in addition to modifying the overall pipeline definition.}
+
+    \end{itemize}
+
+    One of the goals of the new Butler and SuperTask effort is to minimize the amount of integration effort.
+    When running the exact same configuration as the developer, the new Butler and SuperTask effort could definitely help.
+    However, it is not yet clear how much integration/configuration effort will still be required with the new Butler+SuperTask if the operator wishes to run something different than the default pipeline configuration.
+  }
+
+  \item{
+    At the time of this test, it is not yet understood exactly how the Data Facility would configure the execution of the pipeline for best scalability and management.
+    This execution may not match small-scale developer tests where getting individual results faster is more important.
+    The desire is to have the test match the expected release campaign execution configuration
+  as much as possible.
+    As work is done on both the pipeline science and the Batch Processing Service, effort should be undertaken to determine the release campaign execution configuration.
+  }
+
+\end{itemize}
+
+\paragraph{Infrastructure Issues}
+
+During LDM-503-2 pipeline integration and actual test execution, there were multiple infrastructure issues with the Verification Cluster.
+
+\begin{itemize}
+
+  \item{
+    The Verification Cluster had multiple outages due to networking equipment.
+    The Data Facility's Network Group replaced the broken routers and purchased a hot spare.
+  }
+
+  \item{
+    The Verification Cluster can be very busy making it difficult to acquire nodes to do the test work (both initial configuration test and actual full test).
+    To support production processing, production reservations were added to Slurm, and production
+queues are planned in the future.
+  }
+
+
+  \item{
+    This test used the scratch filesystem for both the Data Backbone as well as the job scratch.
+    This filesystem is regularly full preventing the running of the tests.
+    The system admins turned on the purging of the scratch filesystem, and production filesystems are being purchased and installed in the next cycle as described in the Aquisition Strategy Document \textbf{CITATION NEEDED}.
+  }
+
 \end{itemize}
 
+\paragraph{Butler Issues}
+
+It is expected that the new version of the Butler should solve all of these issues.
```

Comment:
Given that, I'm not sure if we need to include this material here. It's certainly the case that future development should make everything better, but this "suggested improvement" boils down to "we should do the work we're already planning to do".

I'm going to try shortening it and moving it to the "problems encountered" section. Does that work for you @hsinfang , et al?